### DISCUSSION TITLE
todo/2023/june-todo/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#14] [General]
# todo/2023/june-todo/

https://guozheng.netlify.app/todo/2023/june-todo/

**Comment by CyrusYip** [comment]
> 不知道更换博客主题之后，域名.rbind.io是否还能保留。

换主题不影响。原来你申请了 rbind 子域名，我还一直访问以前那个。你还是把 [baseURL](https://github.com/residualsun1/Residualsun/blob/577fb21c96afece1167a8379773e9b2766a6053a/config.yaml#L1) 改成 rbind 子域名吧，不然访问旧站不会跳转过去。

**Reply by residualsun1** [reply]
我说为什么我修改字体为思源宋体后，访问自己博客时有时能加载出来，有时又不行。改回来后刚好解决了这个问题。
